### DISCUSSION TITLE
Cant Install foocus

### DISCUSSION THREAD

**Discussion by blacksatijn** [lllyasviel/Fooocus#2854] [Q&A] [answered]
New to github (please go easy...)  I tried to download focus , everything goes fine and at the end it asks for me to click any key and nothing happens , no screen comes up, nothing , what could i be doing wrong?
im clicking on the run prompt 
Any help would be appreciated 

![image](https://github.com/lllyasviel/Fooocus/assets/125201996/567b9ec3-ce51-46ad-b5fd-1a3acdd3490d)

**Comment by mashb1t** [comment, marked as answer]
seems like your Fooocus didn't update itself correctly. Please restart Fooocus and check, if the version number now correctly reports as 2.3.1 as well as run `git pull` manually in the Fooocus folder (where the config.txt is).

**Reply by blacksatijn** [reply]
Many thanks that helped!
